configure

```diff
@@ -528,11 +531,19 @@ if ($opts{'safety-profile'}) {
 ## Download ACE+TAO
 my $tao_maj = 1;
 if (!$ace_src || !$tao_src) {
-  my $urlbase = 'http://download.ociweb.com/TAO-' . $tao_version . '/';
-  my $file = 'ACE+TAO-' . $tao_version . '_with_latest_patches_NO_makefiles.'
-    . ($slash eq '/' ? 'tar.gz' : 'zip');
+  my $urlbase;
+  my $file;
+  if ($opts{'doc_group'}) {
+    $urlbase = 'http://download.dre.vanderbilt.edu/previous_versions/';
+    $file = 'ACE+TAO-src-' . $doc_tao_version . '.'
+      . ($slash eq '/' ? 'tar.gz' : 'zip');
+  } else {
+    $urlbase = 'http://download.ociweb.com/TAO-' . $oci_tao_version . '/';
+    $file = 'ACE+TAO-' . $oci_tao_version . '_with_latest_patches_NO_makefiles.'
+      . ($slash eq '/' ? 'tar.gz' : 'zip');
+  }
   $tao_maj = 2 if $opts{'dry-run'};
-  my $download_message = "Downloading ACE+TAO $tao_version with latest patches";
+  my $download_message = "Downloading ACE+TAO $file";
```

Comment:
Please update this to maintain the current message when --doc_group is not used.  The reason for this is that "OCITAO 2.2a" and "OCITAO 2.2a with latest patches" are different versions.